connaisseur/flask_application.py

```diff
@@ -192,6 +192,18 @@ async def __validate_image(
         )
         return
 
+    if (
+        admission_request.operation.upper() == "UPDATE"
+        and admission_request.kind == "ReplicaSet"
+        and admission_request.wl_object._spec.get("replicas", 1) == 0
```

Comment:
Is it possible that the update request also contains other changes, validation of which would be bypassed as long as the request *also* fulfills the replicas condition?

Reply:
Ha, zwei Idioten, [ein Gedanke](https://github.com/sse-secure-systems/connaisseur/pull/1262#discussion_r1341118573) 😁

Reply:
i think your question is adjecent to https://github.com/sse-secure-systems/connaisseur/pull/1262#discussion_r1341118573

should be resolved